instructions.md

```diff
@@ -0,0 +1,89 @@
+# Installing LGTM Enterprise on Google Cloud
+## Preparation
+Throughout this setup guide a few variables are used that are useful to set in advance. Set the Google Cloud project you want to deploy to, the zone to create any zonal resources in and a unique name for this deployment. If you are upgrading an existing installation, set these values to the same values used when LGTM was first set up.
+
+```console
+project="my-project"
+deployment="lgtm"
+zone="us-central1-a"
+```
+
```

Comment:
> *Note: could we use "control pool" rather than "controller"? The reason for asking is that that's the terminology we agreed with Julian way back and is used throughout the help and the Install Guide.*

I'm happy to change it, but it makes less sense with image deployment as there is only ever one controller machine.